### PR TITLE
Basic docker image with jupyter notebook

### PR DESCRIPTION
**Pull request by davemssavage**
Currently includes this bug fix

https://github.com/deepmind/sonnet/pull/6

### CODE REVIEW

**Comment by davemssavage** [comment]
However even with compile fix from @roman3017 sonnet is not working, I see this error message when running the basic sanity check

$ cd ~/
$ python
>>> import sonnet as snt
>>> import tensorflow as tf
>>> snt.resampler(tf.constant([0.]), tf.constant([0.]))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/sonnet/python/ops/resampler.py", line 62, in resampler
    raise ImportError("_gen_resampler could not be imported.")
ImportError: _gen_resampler could not be imported.

Digging in a bit further to resampler.py it appears the _resampler.so file has some errors

```bash
$ python
>>> import sonnet as snt
>>> import tensorflow as tf
>>> _gen_resampler = tf.load_op_library(tf.resource_loader.get_path_to_datafile("/usr/local/lib/python2.7/dist-packages/sonnet/python/ops/_resampler.so"))
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/load_library.py", line 64, in load_op_library
    None, None, error_msg, error_code)
tensorflow.python.framework.errors_impl.NotFoundError: /usr/local/lib/python2.7/dist-packages/sonnet/python/ops/_resampler.so: undefined symbol: _ZN10tensorflow8internal21CheckOpMessageBuilder9NewStringB5cxx11Ev
```

**Comment by davemssavage** [comment]
I signed it!

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,50 @@
+FROM tensorflow/tensorflow:1.0.1
```
Comment (davemssavage) [inline Dockerfile]: I've tried updating to 1.1 but this results in a runtime error when importing sonnet

import sonnet as snt
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/sonnet/__init__.py", line 102, in <module>
    from sonnet.python.ops.resampler import resampler
  File "/usr/local/lib/python2.7/dist-packages/sonnet/python/ops/resampler.py", line 33, in <module>
    tf.resource_loader.get_path_to_datafile("_resampler.so"))
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/load_library.py", line 64, in load_op_library
    None, None, error_msg, error_code)
tensorflow.python.framework.errors_impl.NotFoundError: /usr/local/lib/python2.7/dist-packages/sonnet/python/ops/_resampler.so: undefined symbol: _ZN10tensorflow8internal21CheckOpMessageBuilder9NewStringB5cxx11Ev

updating to 1.1 would remove the need to import the ppa:openjdk-r/ppa repo

Happy to update this patch if a workaround for that can be found.

**Comment by tomhennigan** [comment]
Hi @davemssavage sorry we didn't get back to you here. Development on Sonnet has switched from V1 to V2 and we're closing out old PRs and issues related to V1.

**Pull request closed without merging**